### PR TITLE
Helm chart template landing page

### PR DESCRIPTION
**Pull request by gthuang**
new page

### CODE REVIEW

File: themes/default/content/templates/kubernetes/helm/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    - csharp
+    - yaml
+
+cloud:
+  name: Kubernetes
+  slug: kubernetes
+
+---
+
+The Helm Chart template creates an infrastructure as code project in your favorite language that deploys a Helm chart to an existing cluster using Pulumi. It uses Pulumi's [Native Kubernetes Provider](/registry/packages/kubernetes) to create a new namespace in an existing Kubernetes cluster and a new [Helm chart](/registry/packages/kubernetes/api-docs/helm/v3) that deploys a web application. The template generates a complete infrastructure as code program, including sample application code, to give you a working project out of the box that you can customize easily and extend to suit your needs.
```
Comment (scottslowe) [inline themes/default/content/templates/kubernetes/helm/index.md:27]: I would change "deploys a web application" to "deploys an application (in this case, the Nginx ingress controller)"

File: themes/default/content/templates/kubernetes/helm/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+When the deployment completes, Pulumi exports the following [stack output](/docs/intro/concepts/stack#outputs) values:
+
+name
+: The name of your new Helm chart deployment.
+
+Output values like these are useful in many ways, most commonly as inputs for other stacks or related cloud resources.
+
+## Customizing the project
+
+You can change the behavior of the Helm chart by passing values to it and an example is already included in the template. Projects created with the Helm Chart template expose the following [configuration](/docs/intro/concepts/config) settings:
```
Comment (scottslowe) [inline themes/default/content/templates/kubernetes/helm/index.md:56]: I'd suggest moving this first sentence into a separate paragraph after the last code block in this section (before the "Tidying up") header. 

Also, I'd reword it to something like: "Further customization of the Helm chart itself is accomplished by passing values to the Helm chart in your Pulumi code. An example of passing a few values to the Helm chart is already included in the template for easy reference." (or similar)

**Review by scottslowe — CHANGES_REQUESTED** [review]
Only a few small changes. Also, I noted that the right-hand column says that the template is also available for AWS and Azure, which doesn't seem right, so we should look into that.

File: themes/default/content/templates/kubernetes-application/_index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Kubernetes Application Templates"
+layout: overview
+meta_desc: Pulumi program templates that make it easy to deploy kubernetes applications on AWS, Azure, or Google Cloud Platform.
```
Comment (cnunciato) [inline themes/default/content/templates/kubernetes-application/_index.md:4]:
```suggestion
meta_desc: Pulumi program templates that make it easy to deploy Kubernetes applications on AWS, Azure, or Google Cloud Platform.
```

**Review by scottslowe — APPROVED** [review]

**Review by cnunciato — APPROVED** [review]
Looks good! Only one little thing I noticed. Feel free to merge once the diagrams are in place and https://github.com/pulumi/templates/pull/439 has been merged as well.
